sklego/model_selection.py

```diff
@@ -260,7 +260,7 @@ def KlusterFoldValidation(**kwargs):
 class ClusterFoldValidation:
     """Cross validator that creates folds based on provided cluster method.
     This ensures that data points in the same cluster are not split across different folds.
-    
+
```

Comment:
Ach so! Thank you for explaining, I also fell for this little test tube in the corner. Just to make sure, the "example" also allows for collapsing, isn't?
Waiting for your decision and will adjust it accordingly

Reply:
Yes, for "example" it is just a matter of default (expanded vs collapsed by default).
Let's wait for Vincent feedback/preference to weight on this 🙃

Reply:
Maybe go for usage instead?

Reply:
I thought `Usage` keyword was google-style specific, but at least from the [griffe docs](https://mkdocstrings.github.io/griffe/docstrings/) it seems it is neither google or numpy.

At the end of the day I believe the only difference is that keywords get **bold**-ed (example in screenshot).

I think for now the safest way to proceed is to use `Examples` keyword as it would have the lowest impact and require less changes.

![image](https://github.com/koaning/scikit-lego/assets/42817048/b0db785a-09ca-4a46-9b1f-a09cd4f4c816)

Reply:
Okay, I'll do Examples then, if you change your mind, just ping me